src/components/widgets/sales-growth/sales-growth.directive.coffee

```diff
@@ -26,10 +28,12 @@ module.controller('WidgetSalesGrowthCtrl', ($scope, $q, ChartFormatterSvc, $filt
   $scope.isDataQuantity = true
   w.initContext = ->
     if $scope.isDataFound = angular.isDefined(w.content) && !_.isEmpty(w.content.summary) && !_.isEmpty(w.content.dates)
-
+      
       $scope.productOptions = _.flatten(_.map(w.content.summary, (product) ->
-        return {label: product.code, value: product.id}
+        label = if w.content.organizations.length = 1 then product.name else product.company + ' - ' + product.name
```

Comment:
The test on threshold should also be applied here. Otherwise, you may end-up with products with a huge name directly in the <select>. Unless maybe the name is automatically truncated when you use select?